Core/MIPS/MIPSInt.cpp

```diff
@@ -151,12 +151,7 @@ namespace MIPSInt
 
 	void Int_Break(MIPSOpcode op)
 	{
-		Reporting::ReportMessage("BREAK instruction hit");
```

Comment:
Hm, this reporting is probably still useful.

-[Unknown]

Reply:
Yeah, fair enough. Although our report UI doesn't seem to allow me to search for these...